### PR TITLE
Style Guide fixes, typos, and fixed links

### PR DESCRIPTION
**Pull request by josh-mccrowell-braze**

### CODE REVIEW

**Comment by josh-mccrowell-braze** [comment]
@davidbielik : Would you check out this PR from a technical point of view. I think SDK Authentication is probably the only edited article that needs your consideration. 

@lydia-xie : Would you please review for style purposes? 

Thank you both!

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 1]
```diff
@@ -149,17 +149,17 @@ Currently, SDK Authentication must be enabled as part of initializing the SDK in
...
+Keep in mind that [`changeUser`][11] should only be called when the User ID has _actually changed_. You should not use this method as a way to update the signature if the user ID has not changed.
 {% endalert %}
 
 {% tabs %}
 {% tab Javascript %}
-Supply the JWT Token when calling `changeUser`:
+Supply the JWT Token when calling [`changeUser`][11]:
 
 ```javascript
 import * as braze from"@braze/web-sdk";
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:165]:
```suggestion
import * as braze from "@braze/web-sdk";
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 2]
```diff
@@ -263,29 +263,29 @@ These callback methods are a great place to add your own monitoring or error-log
 {% tab Javascript %}
 ```javascript
 import * as braze from"@braze/web-sdk";
-braze.subscribeToSdkAuthenticationFailures((authFailure) => {
+braze.subscribeToSdkAuthenticationFailures((someErrorEvent) => {
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:266]:
```suggestion
braze.subscribeToSdkAuthenticationFailures((errorEvent) => {
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 3]
```diff
@@ -263,29 +263,29 @@ These callback methods are a great place to add your own monitoring or error-log
...
   // TODO: Optionally log to your error-reporting service
-  // TODO: Check if the errorEvent user matches the currently logged-in user
-  const updated_jwt = await getNewTokenSomehow(errorEvent);
+  // TODO: Check if the someErrorEvent user matches the currently logged-in user
+  const updated_jwt = await getNewTokenSomehow(someErrorEvent);
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:269]:
```suggestion
  const updated_jwt = await getNewTokenSomehow(errorEvent);
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 4]
```diff
@@ -263,29 +263,29 @@ These callback methods are a great place to add your own monitoring or error-log
 {% tab Javascript %}
 ```javascript
 import * as braze from"@braze/web-sdk";
-braze.subscribeToSdkAuthenticationFailures((authFailure) => {
+braze.subscribeToSdkAuthenticationFailures((someErrorEvent) => {
   // TODO: Optionally log to your error-reporting service
-  // TODO: Check if the errorEvent user matches the currently logged-in user
-  const updated_jwt = await getNewTokenSomehow(errorEvent);
+  // TODO: Check if the someErrorEvent user matches the currently logged-in user
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:268]:
```suggestion
  // TODO: Check if the user_id within the `errorEvent` matches the currently logged-in user
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md (hunk 2 above)
Comment (davidbielik) [reply _docs/_developer_guide/platform_wide/sdk_authentication.md:266]: these 3 suggestions should be applied to other code samples below.

**Review by davidbielik — DISMISSED** [review]
left a few comments in SDK Auth

File: _docs/_developer_guide/platform_wide/analytics_overview.md [hunk 5]
```diff
@@ -198,7 +198,7 @@ The following table describes available segmentation options for time attributes
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
 #### Numbers {#integers}
-Numeric attributes have a wide variety of use-cases. Incrementing number custom attributes are useful for storing the number of times a given action or event has occurred. Standard numbers have all sorts of usages, for example: recording shoe size, waist size, or number of times a user has viewed a certain product feature or category.
+Numeric attributes have a wide variety of use cases. Incrementing number custom attributes are useful for storing the number of times a given action or event has occurred. Standard numbers have all sorts of usages, for example: recording shoe size, waist size, or number of times a user has viewed a certain product feature or category.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/analytics_overview.md:201]:
```suggestion
Numeric attributes have a wide variety of use cases. Incrementing number custom attributes are useful for storing the number of times a given action or event has occurred. Standard numbers have all sorts of usages, such as recording shoe size, waist size, or the number of times a user has viewed a certain product feature or category.
```

File: _docs/_developer_guide/platform_wide/analytics_overview.md [hunk 6]
```diff
@@ -215,7 +215,7 @@ The following table describes available segmentation options for numeric attribu
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
 #### Booleans (true/false)
-Boolean attributes are useful for storing subscription statuses, and other simple binary data about your users. The input options that we provide allow you to find users that have explicitly had a variable set to a true/false value in addition to those that don't have any record of that attribute recorded yet.
+Boolean attributes are useful for storing subscription statuses and other simple binary data about your users. The input options that we provide allow you to find users that have explicitly had a variable set to a true/false value in addition to those that don't have any record of that attribute recorded yet.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/analytics_overview.md:218]:
```suggestion
Boolean attributes are useful for storing subscription statuses and other simple binary data about your users. The input options that we provide allow you to find users that have explicitly had a variable set to a boolean in addition to those that don't have any record of that attribute recorded yet.
```

File: _docs/_developer_guide/platform_wide/analytics_overview.md [hunk 7]
```diff
@@ -249,7 +249,7 @@ The following table describes available segmentation options for purchase events
 >  If you would like to segment on the number of times a specific purchase has occurred, you should also record that purchase individually as an [incrementing custom attribute][12].
 
 ## Taxi/ride-sharing app use case {#example-case}
-For this example case, let's consider a taxi/ride-sharing app (such as Hailo, Lyft, etc.) that wants to decide what user data to collect. The following questions and brainstorming process are a great model for marketing and development teams to follow. By the end of this exercise, both teams should have a solid understanding of what custom events and attributes make sense to collect in order to help meet their goal.
+For this example case, let's consider a taxi/ride-sharing app (such as Hailo or Lyft) that wants to decide what user data to collect. The following questions and brainstorming process are a great model for marketing and development teams to follow. By the end of this exercise, both teams should have a solid understanding of what custom events and attributes make sense to collect in order to help meet their goal.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/analytics_overview.md:252]:
```suggestion
For this example, let's consider a ride-sharing app that wants to decide what user data to collect. The following questions and brainstorming process are a great model for marketing and development teams to follow. By the end of this exercise, both teams should have a solid understanding of what custom events and attributes make sense to collect in order to help meet their goal.
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 8]
```diff
@@ -62,7 +62,7 @@ When generating the JWT, the following fields are expected:
 
 | Field | Required | Description                                                                            |
 | ----- | -------- | -------------------------------------------------------------------------------------- |
-| `sub` | **Yes**  | The "subject" should equal the User ID you supply Braze SDKs when calling `changeUser` |
+| `sub` | **Yes**  | The "subject" should equal the User ID you supply Braze SDK when calling `changeUser`  |
 | `exp` | **Yes**  | The "expiration" of when you want this token to expire.                                |
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:66]:
```suggestion
| `sub` | Yes  | The "subject" should equal the User ID you supply Braze SDK when calling `changeUser`  |
| `exp` | Yes | The "expiration" of when you want this token to expire.                                |
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 9]
```diff
@@ -249,11 +249,11 @@ braze.setSdkAuthenticationSignature("signature")
...
+You can use [`subscribeToSdkAuthenticationFailures`](https://js.appboycdn.com/web-sdk/latest/doc/modules/braze.html#subscribetosdkauthenticationfailures) to subscribe to be notified when the SDK requests fail for one of these reasons. A callback function contains an object with the relevant [`errorCode`][9], `reason` for the error, the `userId` of the request (if the user is not anonymous), and the authentication `signature` that caused the error. 
 
-This callback includes the User ID for which the request failed, the relevant [Error Code][9], and the failed signature. If _that user_ is still logged in, you can use this callback as an opportunity to request a new JWT from your server and supply Braze's SDK with this new valid token.
+Failed requests will periodically be retried until your app supplies a new valid JWT. If _that user_ is still logged in, you can use this callback as an opportunity to request a new JWT from your server and supply Braze's SDK with this new valid token.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:256]:
```suggestion
Failed requests will periodically be retried until your app supplies a new valid JWT. If that user is still logged in, you can use this callback as an opportunity to request a new JWT from your server and supply Braze's SDK with this new valid token.
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 10]
```diff
@@ -323,49 +323,39 @@ func handle(_ errorEvent: ABKSdkAuthenticationError?) {
...
-| `reason` | A description of why the request failed. |
-| `error_code` | An internal error code used by Braze. |
-| `user_id` | The user ID from which the request failed. |
-| `signature` | The JWT that failed.|
-{: .reset-td-br-1 .reset-td-br-2}
-
 ## Adding public keys {#key-management}
 
-In the **Manage Settings** page of the dashboard, add your Public Key to a specific app in the Braze dashboard. Each app supports up to 3 Public Keys. Note that the same Public/Private keys may be used across apps.
+In the **Manage Settings** page of the dashboard, add your public key to a specific app in the Braze dashboard. Each app supports up to three public keys. Note that the same public/private keys may be used across apps.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:337]:
```suggestion
In the **Manage Settings** page of the dashboard, add your public key to a specific app in the Braze dashboard. Each app supports up to three public keys. Note that the same public and private keys may be used across apps.
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 11]
```diff
@@ -323,49 +323,39 @@ func handle(_ errorEvent: ABKSdkAuthenticationError?) {
...
 
-To delete a key, or to promote a key to the Primary key, choose the corresponding action in the overflow menu next to each key.
+To delete a key, or to promote a key to the primary key, choose the corresponding action in the overflow menu next to each key.
 
 ## Enabling in the Braze dashboard {#braze-dashboard}
 
 Once your [Server-side Integration][1] and [SDK Integration][2] are complete, you can begin to enable this feature for those specific apps.
 
-Keep in mind, SDK requests will continue to flow as usual - without authentication - _unless_ the app's SDK Authentication setting is switched to **required** in the Braze dashboard.
+Keep in mind, SDK requests will continue to flow as usual - without authentication - _unless_ the app's SDK Authentication setting is set to **Required** in the Braze dashboard.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:352]:
```suggestion
Keep in mind that SDK requests will continue to flow as usual without authentication unless the app's SDK Authentication setting is set to **Required** in the Braze dashboard.
```

File: _docs/_developer_guide/platform_wide/sdk_authentication.md [hunk 12]
```diff
@@ -323,49 +323,39 @@ func handle(_ errorEvent: ABKSdkAuthenticationError?) {
...
 
-Should anything go wrong with your integration (i.e., your app is incorrectly passing tokens to the SDK, or your server is generating invalid tokens), simply **disable** this feature in the Braze dashboard and data will resume to flow as usual, without verification.
+Should anything go wrong with your integration (i.e., your app is incorrectly passing tokens to the SDK, or your server is generating invalid tokens), simply disable this feature in the Braze dashboard and data will resume to flow as usual, without verification.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/sdk_authentication.md:354]:
```suggestion
Should anything go wrong with your integration (i.e., your app is incorrectly passing tokens to the SDK, or your server is generating invalid tokens), disable this feature in the Braze dashboard, and data will resume to flow as usual without verification.
```

File: _docs/_developer_guide/platform_wide/tv_and_ott.md [hunk 13]
```diff
@@ -215,7 +215,7 @@ To target OTT apps for messaging, we recommend creating a segment specific to yo
 
 ## Headless UI {#custom-ui}
 
-For platforms that support in-app messages or Content Cards via headless UI, Braze will deliver a data model (i.e. JSON) that your app can read and use within a UI your app controls. These platforms don't include any out-of-the-box UI or Views.
+For platforms that support in-app messages or Content Cards via headless UI, Braze will deliver a data model (i.e., JSON) that your app can read and use within a UI your app controls. These platforms don't include any out-of-the-box UI or Views.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_wide/tv_and_ott.md:218]:
```suggestion
For platforms that support in-app messages or Content Cards via headless UI, Braze will deliver a data model (i.e., JSON) that your app can read and use within a UI your app controls. These platforms don't include any out-of-the-box UI or views.
```
Not sure if "Views" should be capitalized here, but also I'm not sure what this is referring to... 🤔

**Review by lydia-xie — CHANGES_REQUESTED** [review]
Looks great! Left a few suggestions for your review!

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**